setup.py

```diff
@@ -17,6 +17,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     install_requires=requirements,
+    python_requires=">=3.8",
```

Comment:
Do we actually need py38 because of something?

Reply:
Mainly typing (Literal, TypedDict). I guess we can use this instead:
```
from __future__ import annotations
````